contracts/ProjectFactory.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.2;
+
+import "./Playground.sol";
+
+// Playground is for creating a Project contract for our users
+contract ProjectFactory {
+    function createProject(
+        string calldata title,
```

Comment:
Let's take symbol and name for the new contract.

Reply:
yeah. we can take those two params from users instead of reusing title for both.